aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -263,6 +286,12 @@ module AptosFramework::Coin {
         }
     }
 
+    fun initialize_coin_events(account: &signer) {
+        move_to(account, CoinEvents {
+            register_events: Event::new_event_handle<RegisterEvent>(account),
+        });
+    }
+
```

Comment:
not sure it is worth the division here, would probably just inline.

Reply:
Updated.